src/OMSimulatorLib/OMSimulator.cpp

```diff
@@ -372,6 +372,24 @@ oms_status_enu_t oms3_addBus(const char *cref)
   return system->addBus(tail);
 }
 
+oms_status_enu_t oms3_addTLMBus(const char *cref, const char *domain, const int dimensions, const oms_tlm_interpolation_t interpolation)
+{
+  //todo Fix!
```

Comment:
@robbr48 What needs to be fixed here?

Reply:
Nothing actually, I just forgot to remove the comment :)